staging/src/k8s.io/apimachinery/pkg/util/mergepatch/util.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package mergepatch
 
 import (
+	"bytes"
+	"encoding/json"
```

Comment:
Should use "apimachinery/pkg/util/json"
Using this json pkg can avoid some number conversion issue.